### DISCUSSION TITLE
Requesting a multiline (potentially markdown) column type

### DISCUSSION THREAD

**Discussion by tlhunter** [community/community#10211] [Projects and Issues]
For my use-case I'm using the new project feature as a CRM to track customers. The Status concept maps nicely to their position in the sales pipeline (prospect, paying, churn, etc). I can make a card view grouped by status to see customers overall in the sales pipeline. I can also see an alphabetical tabular list for quickly finding a specific customer.

One thing I need to do that doesn't map cleanly (but can be achieved using the old project feature) is tracking when I have meetings with customers. In the previous project interface I would use a `<details>` block and a bullet list of dates and quick summary of the meeting. In the new project interface if I paste newline text they seem to be converted into a single line.

Ideally, if enabled for a card view, such a column would display newlines and applicable text. And, if enabled for a table view, would behave just like common spreadsheet software: only the first line is displayed in tabular view, but when selecting the cell, it then displays the newline content.

I believe this is the only feature holding me back from using the new projects feature as a CRM.

**Comment by yegeniy** [comment]
+1 to this or something like it.

"Text" is nice, but also having full "markdown" fields would be nice as well. 

<img width="301" alt="image" src="https://user-images.githubusercontent.com/54481302/159336028-2bd90a41-9d8d-43a8-a8d4-1bac17732b2f.png">

It would be swell if there was also a built-in column for the Issue Description Draft. Similar to the Title column.

I'm looking for something less heavy than the UI currently available by clicking the "Title" column. I'd like to be able to glance over the issue Draft from the spreadsheet view. Or at the very least the first megabyte of it.
